Formula/pyenv.rb

```diff
@@ -45,6 +45,10 @@ def install
     #   - https://github.com/pyenv/pyenv/issues/1056#issuecomment-356818337
     #   - https://github.com/Homebrew/homebrew-core/pull/22727
   end
+  
```

Comment:
```suggestion

```

Reply:
Thanks, this needs a bunch of work and testing. I'll get on it and ping when it's done.